package.json

```diff
@@ -78,6 +78,7 @@
     "lint:fix": "pnpm -r lint:fix",
     "pack:packages": "./scripts/pack.sh",
     "playwright:install": "npx playwright install --with-deps",
+    "scripts:generate-cloudbuild": "bun ./tooling/cloudbuild/generateCloudbuild.ts",
```

Comment:
@geoffgscott I know you don't like these scripts but I think this warrants a new one

Reply:
If we want to use bun prefer the workspace level version `pnpm bun run ...`
Or we can use the workspace vite-node

Reply:
Yeah, we use vite-node everywhere else, can we use that here, too?